pdr_backend/util/predictoor_stats.py

```diff
@@ -0,0 +1,128 @@
+from typing import List, Dict, Tuple, Union
+from pdr_backend.models.prediction import Prediction
+
+PredictionDetail = Tuple[str, str, str]
+PredictoorStats = Dict[str, Union[str, float, int, List[PredictionDetail]]]
+PairTimeframeStats = Dict[str, Union[str, float, int]]
+StatisticsResult = Dict[
+    str, Union[float, List[PredictoorStats], List[PairTimeframeStats]]
+]
+
+
+def get_pure_stats(all_predictions: List[Prediction]):
+    stats: Dict[str, Dict] = {"pair_timeframe": {}, "predictoor": {}}
+    correct_predictions = 0
```

Comment:
Why is `correct_predictions` apart from `stats`? Isn't it a stat too?

And, why have this all as a return dict of `stats` versus returning a tuple of `(correct_predictions, pair_timeframe, predictoor_stats)`?